generators/csharp/dynamic-snippets/src/__test__/__snapshots__/DynamicSnippetsGenerator.test.ts.snap

```diff
@@ -373,54 +431,67 @@ exports[`snippets (default) > nullable > 'Query parameters' 1`] = `
 using Acme.Core;
 using Acme.Nullable;
 
-var client = new AcmeClient{
-    clientOptions = new ClientOptions{
+var client = new AcmeClient(
+    clientOptions: new ClientOptions{
         BaseUrl = "https://api.example.com"
     }
-}
-await client.Nullable.GetUsersAsync(new GetUsersRequest{
-    Usernames = new List<string?>(){
-        "john.doe",
-        "jane.doe",
-    },
-    Tags = new List<string?>(){}
-}).ConfigureAwait(false)"
+);
+
+await client.Nullable.GetUsersAsync(
+    new GetUsersRequest{
+        Usernames = new List<string?>(){
+            "john.doe",
+            "jane.doe",
+        },
+        Tags = new List<string?>(){}
+    }
+);
+"
 `;
 
 exports[`snippets (default) > single-url-environment-default > 'Production environment' 1`] = `
 "using Acme;
 using Acme.Core;
 
-var client = new AcmeClient{
-    Token = "<YOUR_API_KEY>", clientOptions = new ClientOptions{
+var client = new AcmeClient(
+    token: "<YOUR_API_KEY>",
+    clientOptions: new ClientOptions{
         BaseUrl = AcmeEnvironment.Production
     }
-}
-await client.Dummy.GetDummyAsync().ConfigureAwait(false)"
+);
+
+await client.Dummy.GetDummyAsync();
+"
 `;
 
 exports[`snippets (default) > single-url-environment-default > 'Staging environment' 1`] = `
 "using Acme;
 using Acme.Core;
 
-var client = new AcmeClient{
-    Token = "<YOUR_API_KEY>", clientOptions = new ClientOptions{
+var client = new AcmeClient(
+    token: "<YOUR_API_KEY>",
+    clientOptions: new ClientOptions{
         BaseUrl = AcmeEnvironment.Staging
     }
-}
-await client.Dummy.GetDummyAsync().ConfigureAwait(false)"
+);
```

Comment:
Should be 
```csharp
var client = new AcmeClient(
    token: "<YOUR_API_KEY>",
    clientOptions: new ClientOptions
    {
        BaseUrl = AcmeEnvironment.Staging
    }
);
```
or
```csharp
var client = new AcmeClient(
    token: "<YOUR_API_KEY>",
    clientOptions: new ClientOptions { BaseUrl = AcmeEnvironment.Staging }
);
```

Reply:
I made this change, but it impacts the `( ... )` use case since they're so tightly coupled as-is. I'd vote to come back to this improvement when we get around to resolving [this thread](https://github.com/fern-api/fern/pull/6326#discussion_r1985479592).